tests/Unit/Test_TestingFramework.cpp

```diff
@@ -6,7 +6,7 @@
 #include "Parallel/Abort.hpp"
 #include "tests/Unit/TestingFramework.hpp"
 
-SPECTRE_TEST_CASE("TestFramework.Approx", "[Unit]") {
+SPECTRE_TEST_CASE("Test.TestingFramework.Approx", "[Unit]") {
```

Comment:
"Unit.TestingFramework.Approx"  and squash into previous commit

Reply:
The tests in `tests/Unit/Test_TestHelpers.hpp` are named `"Test.TestHelpers.blah"`. Should I change those to `"Unit.TestHelpers.blah"` as well?